### PR TITLE
Update governance page for clarity

### PR DESCRIPTION
**Pull request by communikit**
- Page updated to highlight contribution policy
- Questions moved to FAQ section
- Editorial updates

See http://b/181798269

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
 supporting votes from core contributors on the mailing list and no veto within four business days.
 We expect that new contributors will submit a number of patches before they become core
 contributors.
 
-## Accepting Contributions
+Contact the core team at <a href="mailto:bazel-core@googlegroups.com">
+bazel-core@googlegroups.com</a>.
 
-Please also see our [contribution guidelines](contributing.html).
+The Bazel project also accepts individual contributions from external
```
Comment (floriographygoth) [inline governance.md:22]: I think this should be in the Contribution policy section, perhaps?

File: governance.md [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
-# Governance
+# Contribution policy
 
-The Bazel project is led by a core group of contributors, initially Googlers, and managed by the
-community. The group of core contributors is self-managing - core contributors are added by two
+This page covers Bazel's governance model and contribution policy.
+
+## Governance model
+
+The Bazel project is led by a core group of contributors,  and managed by the
```
Comment (floriographygoth) [inline governance.md:12]: Delete the comma and one of the spaces after contributors

**Review by jin — APPROVED** [review]

File: governance.md [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
+community. At this time, all core contributors work for Google, but this will hopefully
```
Comment (floriographygoth) [inline governance.md:13]: Edit out future tense.

File: governance.md [hunk 4]
```diff
@@ -1,27 +1,30 @@
...
+change over time. The group of core contributors is self-managing - core contributors are added by two
 supporting votes from core contributors on the mailing list and no veto within four business days.
 We expect that new contributors will submit a number of patches before they become core
 contributors.
 
-## Accepting Contributions
+Contact the core team at <a href="mailto:bazel-core@googlegroups.com">
+bazel-core@googlegroups.com</a>.
 
-Please also see our [contribution guidelines](contributing.html).
```
Comment (floriographygoth) [inline governance.md:16]: I think it's helpful to have a link to the contribution guidelines. Perhaps after the list have it as a standalone paragraph that says "For more details on contributing to Bazel, see our [contribution guidelines](contributing.html)."

**Review by floriographygoth — COMMENTED** [review]
Some starting points.

File: governance.md (hunk 2 above)
Comment (communikit) [reply governance.md]: Done.

File: governance.md (hunk 3 above)
Comment (communikit) [reply governance.md]: Removed the future speculation part.

File: governance.md (hunk 4 above)
Comment (communikit) [reply governance.md:16]: Done.

File: governance.md (hunk 1 above)
Comment (communikit) [reply governance.md:24]: Added it to the beginning.

**Pull request merged**